.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+  "bracketSpacing": true,
+  "jsxBracketSameLine": false,
+  "printWidth": 100,
+  "semi": true,
+  "singleQuote": true,
+  "trailingComma": "all",
+  "arrowParens": "avoid"
```

Comment:
Default in 2.0.0 is `always`. This would introduce 600 changed filed. I guess we already agreed to not follow it.

Reply:
I would be in favor of using the default
```suggestion
```

Reply:
This makes me so happy :D Let's do this and get rid of the explicit defaults.

Reply:
Well, singleQuote and trailingComma seems the only two we need to keep?  https://github.com/facebook/react/blob/master/.prettierrc.js

Reply:
As long as it matches the format in https://reactjs.org/, we should be fine.